deployment/environment/memory/job_service_client.go

```diff
@@ -149,7 +150,6 @@ func (j *JobServiceClient) ProposeJob(ctx context.Context, in *jobv1.ProposeJobR
 	if err != nil {
 		return nil, fmt.Errorf("failed to propose job: %w", err)
 	}
-	fmt.Printf("proposed job uuid %s with id, spec, version: %d\n%s\n%d\n", extractor.ExternalJobID, appProposalID, in.Spec, len(proposals)+1)
```

Comment:
intentional?

Reply:
Do we need this output? I can put it back.